docs/release_notes/v1.0.0-rc.2.md

```diff
@@ -0,0 +1,513 @@
+
+First run these two commands to disable `helm upgrade` from uninstall 0.11.x placement service.
+
+```sh
+kubectl annotate deployment dapr-placement helm.sh/resource-policy=keep -n dapr-system
+kubectl annotate svc dapr-placement helm.sh/resource-policy=keep -n dapr-system
+```
+
+Then, export certs:
+
+```sh
+dapr mtls export -o ./certs
+```
+
+Now, upgrade Dapr to 1.0.0-rc.2 using the below commands:
+
+```sh
+helm repo update
+helm upgrade dapr dapr/dapr --version 1.0.0-rc.2 --namespace dapr-system --reset-values --set-file dapr_sentry.tls.root.certPEM=./certs/ca.crt --set-file dapr_sentry.tls.issuer.certPEM=./certs/issuer.crt --set-file dapr_sentry.tls.issuer.keyPEM=./certs/issuer.key --set global.ha.enabled=true --wait
+```
+
+Due to Helm being unable to upgrade CRDs, we will need to upgrade CRD directly.
+```sh
+kubectl replace -f https://raw.githubusercontent.com/dapr/dapr/21636a9237f2dcecd9c80f329e99b512e8377739/charts/dapr/crds/configuration.yaml
+kubectl replace -f https://raw.githubusercontent.com/dapr/dapr/21636a9237f2dcecd9c80f329e99b512e8377739/charts/dapr/crds/components.yaml
+```
+
+Once Dapr is installed completely, ensure that 0.11.x dapr-placement service is still running and **wait until all PODs are running**
+```sh
+kubectl get pods -n dapr-system -w
+
+NAME                                     READY   STATUS    RESTARTS   AGE
+dapr-dashboard-69f5c5c867-mqhg4          1/1     Running   0          42s
+dapr-operator-5cdd6b7f9c-9sl7g           1/1     Running   0          41s
+dapr-operator-5cdd6b7f9c-jkzjs           1/1     Running   0          29s
+dapr-operator-5cdd6b7f9c-qzp8n           1/1     Running   0          34s
+dapr-placement-5dcb574777-nlq4t          1/1     Running   0          76s  ---- 0.11.x placement
+dapr-placement-server-0                  1/1     Running   0          41s
+dapr-placement-server-1                  1/1     Running   0          41s
+dapr-placement-server-2                  1/1     Running   0          41s
+dapr-sentry-84565c747b-7bh8h             1/1     Running   0          35s
+dapr-sentry-84565c747b-fdlls             1/1     Running   0          41s
+dapr-sentry-84565c747b-ldnsf             1/1     Running   0          29s
+dapr-sidecar-injector-68f868668f-6xnbt   1/1     Running   0          41s
+dapr-sidecar-injector-68f868668f-j7jcq   1/1     Running   0          29s
+dapr-sidecar-injector-68f868668f-ltxq4   1/1     Running   0          36s
+```
+
+
+Restart your application deployments to update the Dapr runtime.
+```sh
+kubectl rollout restart deploy/<DEPLOYMENT-NAME>
+```
+
+Once the deployment is completed, delete the 0.11.x dapr-placement service by running the following commands:
+```sh
+kubectl delete deployment dapr-placement -n dapr-system
+kubectl delete svc dapr-placement -n dapr-system
+```
+
+All done!
+
+##### 2. Upgrade from existing cluster running 1.0.0-rc.1
+
+First, export certs:
+
+```sh
+dapr mtls export -o ./certs
+```
+
```

Comment:
```suggestion

Now, check if you have any exporter type components, by running the following command `kubectl get components -o=custom-columns='NAME:metadata.name,TYPE:spec.type'` and look for components with type `exporter.*`.

Since exporter components are no longer supported, you will need to remove them prior to upgrading. You may want to print their content for your reference using `kubectl get component <component-name> -o yaml`.

```